### PR TITLE
move menu to horizontal

### LINKED ISSUE #187: Move the restart build button out of the hamburger menu
moved from https://github.com/drone/drone/issues/2278

> The restart build button is in a very unintuitive location at present.
>
> Discussed on https://discourse.drone.io/t/drone-0-8-no-restart-button/908

### PR DESCRIPTION
**Pull request by kwolfy**
I suggest moving the menu to a horizontal area to improve usability. It's actually resolve #187 too. 
Screenshots:
img2
img1

### CODE REVIEW

File: src/screens/repo/screens/build/menu.js [hunk 1]
```diff
@@ -58,31 +58,28 @@ export default class BuildMenu extends Component {
 
 		const hideCancel = assertBuildMatrix(build) && !proc;
 
+		const rightSide = !build ? (
+			undefined
+		) : (
+			<section>
+				{build.status === "peding" ||
```
Comment (tboerger) [inline src/screens/repo/screens/build/menu.js]: Typo? Should be pending.

Comment (kwolfy) [reply src/screens/repo/screens/build/menu.js]: I think so, but it was in the master
https://github.com/drone/drone-ui/blob/master/src/screens/repo/screens/build/menu.js#L69

 In any case I fixed in last commit

**Comment by tboerger** [comment]
Is there anything left on the burger menu?

**Comment by kwolfy** [comment]
@tboerger Yes, there are repositories, token, logout
img5

**Comment by laszlocph** [comment]
I'm using this in my fork for 2 weeks now. Quality stuff. 

Please consider merging it.

**Pull request closed without merging**